corehq/apps/userreports/tasks.py

```diff
@@ -171,3 +173,10 @@ def _run_report(backend_to_use):
 
     objects = experiment.run()
     return objects
+
+
+@task(queue=UCR_INDICATOR_CELERY_QUEUE, ignore_result=True, acks_late=True, retry=True)
```

Comment:
retry=True is not a valid option to pass a task decorator as far as i know. there is a retry process described [here](http://docs.celeryproject.org/en/3.1/userguide/tasks.html#retrying), but the worker lost errors which happen for sigterm, sigsegv, etc. weren't catchable by it the last time i tried